### DISCUSSION TITLE
Original PCB files available?

### DISCUSSION THREAD

**Discussion by MrAureliusR** [frntc/RAD#40] [General]
The 24-SOICW SN74CBT3861DWR is currently marked as **Last Time Buy** on Digikey, along with many other variants. In the future, it appears the standard 24-SSOP SN74CBTD3861DBQR will be available continuously (marked as Active). I wanted to fix this on the PCB and submit it as a pull request so that others could benefit from being able to make the RAD in the future. However, with only the Gerbers available, making changes like this isn't easy. While KiCAD can import Gerbers back into the PCB Editor, it cannot recover info like footprint groupings, fill zones, etc. Everything is simply imported as polygons on the correct layers. 

Is the original PCB file available anywhere? I note that the PCB is licensed under CC instead of an open hardware license, so technically I guess only the artwork is available, but this is actually why I prefer to release things under open hardware licenses. I can't easily make changes to the existing project without re-creating it from scratch in KiCAD, which is possible with the information contained in the Gerbers, albeit a slightly tedious task.

**Comment by frntc** [comment]
Hey, 

I'm aware of the life-time-end of the SOIC-variants. Actually there's a SSOP-RAD-PCB since quite some time which I have prepared for restore-store.de (it's just not released here yet, which I will do at some point). So far the SOIC-ICs are still available and the SOIC-CBTD3861 from Aliexpress/China are confirmed to work well. I assume the SOIC-ICs are also easier to solder for most :-)

For the future there's also a bunch of other FET-level shifters suitable as a replacement (just requires little modification), so we're not running out of SOIC alternatives anyway!

I'm not going to release the KiCAD-projects. There have been too many annoying experiences in the past with people ignoring licenses and trying to make money with my projects. It's detached from your request, but please understand that I don't want to make it any easier for them.

**Reply by MrAureliusR** [reply]
But I mean... the PCB files are available. How would giving access to the KiCAD projects make it any easier for them to create clones? They can already just edit the Gerbers.
Either way, I respect your decision, I just am having trouble understanding it.

**Reply by frntc** [reply]
Well, the full KiCAD project and Gerbers are two different things as you wrote above ;-)
